apps/docs/package.json

```diff
@@ -10,7 +10,8 @@
     "network": "npm run build && vite preview --host",
     "dev": "vite",
     "start": "vite preview --port=3000",
-    "lint": "eslint src/**/*.{ts,vue} --fix"
+    "lint": "eslint . --ext .ts,.vue",
```

Comment:
Ah, i just wanted to make a distinction so people can have the choice when they want to get to their stuff Linted. 

But you are right technically speaking we don't really "need" it, more a nice to have. 

Also used by the pipeline to make sure the Codebase is fine. Since that pipeline doesn't commit fixes back, if say a dev messes up, the pipeline fixes it but never commits it then the fix is just lost, right?